files/project_team/IFCTunnelDep3_Geo_BookA.ifc

```diff
@@ -0,0 +1,663 @@
+#262= IFCLINEARPLACEMENT(#3,#263,$);
+#263= IFCAXIS2PLACEMENTLINEAR(#264,$,$);
+#264= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(3016.177822),$,$,$,#40);
+#265= IFCPRODUCTDEFINITIONSHAPE($,$,(#266));
+#266= IFCSHAPEREPRESENTATION(#205,'Body','BoundingBox',(#267));
+#267= IFCBOUNDINGBOX(#268,0.4,0.4,0.4);
+#268= IFCCARTESIANPOINT((-0.2,-0.2,-0.2));
+#269= IFCRELNESTS('3ERsYcS8rC$e0oJ0oIn3EL',$,'name',$,#25,(#270,#325,#428));
+#270= IFCALIGNMENTHORIZONTAL('2XoMw3ga1FjOE3y_TokcsH',$,$,'VMU',$,#320,#323);
+#271= IFCRELNESTS('0Y3w7yArD6O9QV2GrNT47G',$,'name',$,#270,(#272,#280,#288,#296,#304,#312));
+#272= IFCALIGNMENTSEGMENT('399sQoxCb4nRKWEUHY6qBh',$,'E',$,$,#273,#276,#278);
+#273= IFCLOCALPLACEMENT($,#274);
+#274= IFCAXIS2PLACEMENT3D(#275,$,$);
+#275= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#276= IFCPRODUCTDEFINITIONSHAPE($,$,(#277));
+#277= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#154));
+#278= IFCALIGNMENTHORIZONTALSEGMENT(', code=POB',', code=TS',#279,-3.04005462523801,0.0,0.0,1508.341099,$,.LINE.);
+#279= IFCCARTESIANPOINT((162572.948873,6721939.071596));
+#280= IFCALIGNMENTSEGMENT('2kczufwsTBxgT$0zWl8W7d',$,'E',$,$,#281,#284,#286);
+#281= IFCLOCALPLACEMENT($,#282);
+#282= IFCAXIS2PLACEMENT3D(#283,$,$);
+#283= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#284= IFCPRODUCTDEFINITIONSHAPE($,$,(#285));
+#285= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#162));
+#286= IFCALIGNMENTHORIZONTALSEGMENT(', code=TS',', code=SC',#287,-3.04005461353714,0.0,1900.0,92.0,$,.CLOTHOID.);
+#287= IFCCARTESIANPOINT((161072.376573,6721786.180648));
+#288= IFCALIGNMENTSEGMENT('01Vm47fMTEAfVRC2YJfgEd',$,'E',$,$,#289,#292,#294);
+#289= IFCLOCALPLACEMENT($,#290);
+#290= IFCAXIS2PLACEMENT3D(#291,$,$);
+#291= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#292= IFCPRODUCTDEFINITIONSHAPE($,$,(#293));
+#293= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#169));
+#294= IFCALIGNMENTHORIZONTALSEGMENT(', code=SC',', code=CS',#295,-3.01584409831919,1900.0,1900.0,40.61357,$,.CIRCULARARC.);
+#295= IFCCARTESIANPOINT((160980.931044,6721776.117138));
+#296= IFCALIGNMENTSEGMENT('3AB1h0OCT5$xK8a0vGDIBf',$,'E',$,$,#297,#300,#302);
+#297= IFCLOCALPLACEMENT($,#298);
+#298= IFCAXIS2PLACEMENT3D(#299,$,$);
+#299= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#300= IFCPRODUCTDEFINITIONSHAPE($,$,(#301));
+#301= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#177));
+#302= IFCALIGNMENTHORIZONTALSEGMENT(', code=CS',', code=ST',#303,-2.99446852260906,1900.0,0.0,92.0,$,.CLOTHOID.);
+#303= IFCCARTESIANPOINT((160940.695663,6721770.593252));
+#304= IFCALIGNMENTSEGMENT('11g5Rzs9fDQ8i$rC4D9Ybz',$,'E',$,$,#305,#308,#310);
+#305= IFCLOCALPLACEMENT($,#306);
+#306= IFCAXIS2PLACEMENT3D(#307,$,$);
+#307= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#308= IFCPRODUCTDEFINITIONSHAPE($,$,(#309));
+#309= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#184));
+#310= IFCALIGNMENTHORIZONTALSEGMENT(', code=ST',', code=POE',#311,-2.97025800968906,0.0,0.0,2110.789685,$,.LINE.);
```

Comment:
```suggestion
#310= IFCALIGNMENTHORIZONTALSEGMENT('code=ST','code=POE',#311,-2.97025800968906,0.0,0.0,2110.789685,$,.LINE.);
```

It seems weird that the tags start with `, ´. If this is a quick fix, I would welcome the change.

Reply:
It is not a quick fix since that alignment data comes from the LandXML=>IFC conversion by Peter Bonsma. However, I can commit your proposal here (certainaly looks a lot better :-))